pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -270,6 +291,31 @@ func GetOCIHooksConfigContent(configFile string, profile *performancev2.Performa
 	return outContent.Bytes(), nil
 }
 
+// GetRunCWrapperContent reads and returns the content of the runC wrapper file
+func GetRunCWrapperContent(wrapperFile string, profile *performancev2.PerformanceProfile) ([]byte, error) {
+	content, err := ioutil.ReadFile(wrapperFile)
+	if err != nil {
+		return nil, err
+	}
+
+	// we either use a special sentinel value and we do logic in the wrapper itself, or we pass
+	// through a list of all the online CPUs. We don't know the latter, performance operator has
+	// no hardware awareness to date, so we need to do the former.
+	cpuList := "UNRESTRICTED"
```

Comment:
An empty string would also be ok, but that works too

Reply:
yeah. I believe this explicit value is less ambiguous - this is pretty obviously a sentinel value, while "" (empty string) still leave some room for doubt - could it be an interpolation error? bad input?
Anyway is a very tiny difference so if reviewers feel "" is better I'm fine with that as well.